src/hypertable.c

```diff
@@ -1835,6 +1835,11 @@ ts_hypertable_create_internal(PG_FUNCTION_ARGS, bool is_dist_call)
 				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
 				 errmsg("invalid main_table: cannot be NULL")));
 
+	if (migrate_data && is_dist_call)
+		ereport(ERROR,
+				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
```

Comment:
nit: it's up to you, but maybe use FEATURE_NOT_SUPPORTED as an error code instead?

Reply:
It is better, so switched.